compose-ui/src/main/java/cm/ben/android/bazel/compose/example/ui/MainActivity.kt

```diff
@@ -17,3 +18,7 @@ class MainActivity : AppCompatActivity() {
 
 @Composable
 fun Greeting(name: String) = Text(text = "Hello $name!")
+
+@Preview
```

Comment:
I can remove this if you want... it doesn't work right now 🤷

Reply:
hi @alexjlockwood - any idea what is the status of the preview feature in Bazel Android projects? and if it is working, what are the necessary steps to use it?